README.md

```diff
@@ -4,3 +4,49 @@ Consists of:
 
 1. A Node.js server for serving marketing messages to theguardian.com ([dotcom-rendering](https://github.com/guardian/dotcom-rendering/)). See [server.md](./docs/server.md) for details
 2. An npm package with types and logic for use by dotcom-rendering.
+
+### Set Up
+
+To get the server running on local, just perform the following operations
+
+- Clone the repository
+
+```
+$ git clone https://github.com/guardian/support-dotcom-components
+$ cd support-dotcom-components
+```
+
+- Ensure that you are running the right version of node
+
+```
+$ nvm install
+$ nvm use
+```
+
+(`nvm install` is only required if you didn't already have the right version installed on your machine.)
+
+- We are using pnpm as package manager. If you do not have it installed on your machine, please install it. Then
+
+```
+pnpm install
+pnpm build-server
+pnpm prestart
```

Comment:
prestart seems to effective duplicate start-watch which is called with pnpm start - is this needed?